src/forder.c

```diff
@@ -430,13 +430,15 @@ SEXP forder(SEXP DT, SEXP by, SEXP retGrpArg, SEXP sortGroupsArg, SEXP ascArg, S
     if (!isVectorAtomic(DT)) error("Input is not either a list of columns, or an atomic vector.");
     if (!isNull(by)) error("Input is an atomic vector (not a list of columns) but by= is not NULL");
     if (!isInteger(ascArg) || LENGTH(ascArg)!=1) error("Input is an atomic vector (not a list of columns) but ascArg= is not a length 1 integer");
```

Comment:
in those few lines above you could move `error()` calls into new line so code coverage can test them

Reply:
Thanks, will do.  I sometimes forget this difference between R and C coverage (R coverage does detect uncovered branches within a line, while C coverage doesn't).